doc/content/shortcodes.md

```diff
@@ -3,4 +3,115 @@ title: Shortcodes
 sidebar: true
 ---
 
```

Comment:
I think we are missing a one liner to say what are these. When you are new to Hugo, it's not immediately clear where/how to use these.

Reply:
In b87e168, I added a short paragraph to the top of the file by editing `tools/render_shortcode_docs.py`.

@stefanv I also removed the old stub file for `doc/content/shortcodes.md`.  Instead I added some text in `tools/render_shortcode_docs.py` that will be added as a comment to the top of the  `doc/content/shortcodes.md` that says:
```
<!-- DO NOT EDIT.  This file is generated by `tools/render_shortcode_docs.py`.  -->
```

Reply:
Perfect, thanks!